corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -276,40 +281,47 @@ <h4 class="modal-title" data-bind="visible: !$root.editColumnController.isNew()"
                                      }" ></select>
                 </div>
               </div>
-              <!-- ko if: $root.editColumnController.showColumnFilters -->
-                <hr />
-                <div class="form-group"
-                     data-bind="visible: hasFilters">
-                  <div class="datagrid-column-filter-clause">
-                    {% trans "Match" %}
-                    <select data-bind="value: clause,
-                                     event: { change: $root.editColumnController.updateFilter }"
-                            class="form-control">
-                      <option value="all">
-                        {% trans "all of the" %}
-                      </option>
-                      <option value="any">
-                        {% trans "any of the" %}
-                      </option>
-                    </select>
-                    {% trans "following conditions:" %}
-                  </div>
-                </div>
 
+              <hr data-bind="visible: $root.editColumnController.showColumnFilters()
+                             || $root.editColumnController.showColumnFilterPlaceholder()" />
+              <button type="button"
+                      class="btn btn-default disabled"
+                      disabled="disabled"
+                      data-bind="visible: $root.editColumnController.showColumnFilterPlaceholder">
+                {% trans "Filter Data in Column" %}
+              </button>
+
+              <!-- ko if: $root.editColumnController.showColumnFilters -->
                 <!-- ko foreach: appliedFilters -->
-                <div class="datagrid-filter">
+
                   <!-- ko if: $index() === 0 -->
-                  <select class="form-control datagrid-filter-type"
-                          data-bind="options: $root.editColumnController.availableFilterNames,
-                                     optionsText: function (val) {
-                                       return $root.editColumnController.filterTitleByName[val];
-                                     },
-                                     value: filterName,
-                                     event: { change: $root.editColumnController.updateFilterName }"></select>
-                  <!-- /ko -->
-                  <!-- ko if: $index() !== 0 -->
-                  <div class="datagrid-filter-type-spacer"></div>
+                  <div class="datagrid-filter-settings">
+                    {% blocktrans %}
```

Comment:
Is this block easily translatable? I think as a translator this might be a bit confusing.

Reply:
it's weird because the dropdowns are part of the sentence...and if the grammar changes between the languages, that might affect how it reads.

Reply:
that's why I decided to wrap the whole thing in a `blocktrans`, but if that makes it worse then I'll change it up